terraform/HOW_TO.md

```diff
@@ -0,0 +1,43 @@
+# HOW TO
+
+## Deploy in ci
+
+The [deploy-ci](../.github/workflows/deploy-ci.yml) workflow is triggered at each commit on main, and can also be [triggered manually](https://github.com/betagouv/service-national-universel/actions/workflows/deploy-ci.yml)
+
+It builds docker images and deploy only if the unit test are successfull.
+
+## Create a custom environment
+
+The [deploy-custom-env](../.github/workflows/deploy-custom-env.yml) workflow is triggered when a pull request is opened or modified if the corresponding branch name contains **ci-\***, **\*-ci**, or **\*-ci-\***
+It can also be [triggered manually](https://github.com/betagouv/service-national-universel/actions/workflows/deploy-custom-env.yml)
```

Comment:
But you will have to retrigger manually on every new commit if you choose this way